core/src/test/java/org/elasticsearch/action/ingest/SimulatePipelineRequestTests.java

```diff
@@ -89,4 +86,10 @@ public void testSerializationWithXContentBwc() throws IOException {
             }
         }
     }
+
+    public void testUnsupportedReadFrom() {
+        SimulatePipelineRequest request = new SimulatePipelineRequest();
+        Exception exception = expectThrows(UnsupportedOperationException.class,() -> request.readFrom(null));
+        assertThat(exception.getMessage(), equalTo("usage of Streamable is to be replaced by Writeable"));
```

Comment:
I don't think we should be adding a test for each one of these. It is not part of our API (that these are returning UOE).